### PR TITLE
convert the HTML mailer templates to HAML

### PR DESCRIPTION
**Pull request by afeld**
Worth noting that I didn't manually test each of these.

### CODE REVIEW

File: app/views/communicart_mailer/comment_added_email.html.haml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+%table.w-container.main-container{width: '800'}
+  - proposal = @comment.proposal
+  %tr
+    %td.w-container.html-email-message
+      %p
+        Action need on Request \##{proposal.public_identifier} (#{proposal.name}):
```
Comment (cmc333333) [inline app/views/communicart_mailer/comment_added_email.html.haml]: I think we can remove the first pound, as the `public_identifier` includes it, hey?

File: app/views/communicart_mailer/_comments.html.haml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  %table.submodel-section
+    %tr
+      %td.w-container{colspan: 2}
+        %strong Comments
+    - comments.each do |c|
+      %tr{class: ('comment-update' if c.update_comment)}
+      %td.comment-detail.first{valign: 'top'}
+        %p
+          = l(c.created_at.to_date)
+      %td.comment-detail.first{valign: 'top'}
```
Comment (cmc333333) [inline app/views/communicart_mailer/_comments.html.haml]: Looks like "first" was added as a class here?

File: app/views/communicart_mailer/_comments.html.haml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+- if comments.any?
+  %table.submodel-section
+    %tr
+      %td.w-container{colspan: 2}
+        %strong Comments
+    - comments.each do |c|
+      %tr{class: ('comment-update' if c.update_comment)}
```
Comment (cmc333333) [inline app/views/communicart_mailer/_comments.html.haml]: Good; this was one of the issues with the automated conversion

File: app/views/communicart_mailer/_comments.html.haml (hunk 2 above)
Comment (afeld) [inline app/views/communicart_mailer/_comments.html.haml]: Damn, good catch! I think I copied-and-pasted that line.

**Comment by afeld** [comment]
Updated!

**Pull request merged**